src/emcpy/plots/domain.py

```diff
@@ -0,0 +1,107 @@
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
```

Comment:
These will need to be added to `setup.cfg`.
I think that is going to cause headaches (for me).

Reply:
I think it should be okay, all I need to add is `cartopy` based on my python environment.